src/App.js

```diff
@@ -18,6 +19,7 @@ if (IS_ELECTRON) {
   psiturk = new PsiTurk(getTurkUniqueId(), '/complete')
   /* eslint-enable */
 }
+const firebase = true;
```

Comment:
how can this get overwritten to false? should there be an env var?

Reply:
I think I changed that in the latest commit.. maybe I didn't push?

Reply:
I did push.. maybe the PR didn't update :/

Reply:
it says `outdated` here